source/_docs/installation/synology.markdown

```diff
@@ -2,78 +2,287 @@
 layout: page
 title: "Installation on a Synology NAS"
 description: "Instructions to install Home Assistant on a Synology NAS."
-date: 2016-04-16 11:36
+date: 2018-12-05 10:36
 sidebar: true
 comments: false
 sharing: true
 footer: true
 redirect_from: /getting-started/installation-synology/
 ---
 
-<p class='note warning'>
-Synology only provide Python 3.5.1, which is not compatible with Home Assistant 0.65.0 or later. Until Synology offer an updated version of Python, Home Assistant 0.64 is the most recent version that will be able to be installed. You can manually specify the version of Home Assistant to install, for example to install version 0.64.3 you would do `./python3 -m pip install homeassistant==0.64.3`
+<p class="note warning">
+Synology only provide <a href="https://www.synology.com/nl-nl/dsm/packages/py3k">Python 3.5.1</a>, which is not compatible with Home Assistant 0.65.0 or later. Until Synology offer an updated version of Python, Home Assistant 0.64 is the most recent version that will be able to be installed. You can manually specify the version of Home Assistant to install, for example to install version 0.64.3 you would do `./python3 -m pip install homeassistant==0.64.3`
```

Comment:
Please use Markdown to format links
```raw
[Python 3.5.1](https://www.synology.com/nl-nl/dsm/packages/py3k)
```

Reply:
Does Markdown work inside html tags, like Paragraph?
When I try to use Markdown formatted hyperlinks, these are not parsed in the preview.
Instead to compensate, I used hyperlink tags.

In the case Markdown is not supported in this way and html hyperlinks is not accepted, should I remove hyperlinks altogether and only add the URL in "Thread for more info: Python >=3.5.3 on Synology."?
I will not be able to link to the components pages, so should I remove the info about which components for example will work? That info is useful so the user has a idea what this guide fixes, compared to the previous guide.

I could also instead remove the Paragraph tag and just use normal text, but the user might think the warning note is applicable to the new guide on first look (which with a note it catches the eye, so the user would read that first instead of skimming).

Reply:
It should work.